appspec-scripts/stop_app.sh

```diff
@@ -1,9 +1,35 @@
 #!/bin/bash
-set -e
+set -x
```

Comment:
This parameter was removed, for the same reasons like mentioned here: https://github.com/tlsnotary/tlsn/pull/459#discussion_r1532049999
Additionally, CodeDeploy does not flag and stop the deployment in case of ApplicationStop hook failure.

Reply:
i see, wouldn't it be dangerous then if codedeploy doesn't stop if stop_app hook fails to kill the old nightly service, and then start_app also fails to spin up the new nightly due to port still in use, and since start_app has exit 0 at the end, CD will continue, and validate_app will pass also since the old nightly is still running?

Reply:
I can understand your concern. Let me modify the scripts in order to support the **-e** argument :)